app/views/templates/game/index.html.erb

```diff
@@ -0,0 +1,82 @@
+<div class="row">
+  <div class="col l6 offset-l3 s12 z-depth-1">
+    <div class="row">
+      <div class="col s12 red lighten-2">
+        <h1 class="center-align white-text">Codebreaker</h1>
+      </div>
+    </div>
+    <form method="post">
+      <div class="row">
+        <div class="col s12">
+          <p class="center-align">Guess code with number from 1 to 6</p>
+          <div class="input-field col s6 offset-s3">
+            <input id="guess_code" name="guess_code" type="number" class="center-align" max="6666">
+            <label for="guess_code">Guess code</label>
+          </div>
+        </div>
+      </div>
+      <div class="row menu">
+        <div class="col s6 center-align">
+          <button type="submit" class="waves-effect waves-light btn green darken-3"><strong>submit</strong></button>
+        </div>
+        <div class="col s6 center-align">
+            <button formaction="/game/hint" type="submit" class="waves-effect waves-light btn">hint</button>
+        </div>
+      </div>
+    </form>
+    <div class="row">
+      <div class="col s12 center-align">
+        <p>Attempts: <%= @request.game.attempts %>/<%= @request.session[:total_attempts] %></p>
+        <p>Hints: <%= @request.game.hints %>/<%= @request.session[:total_hints] %></p>
```

Comment:
views shouldn't contain logic, things like `<%= @request.game.attempts %>/<%= @request.session[:total_attempts] %>` and ` <%= @request.game.hints %>/<%= @request.session[:total_hints] %>` can be moved to class which will be used like view helper